### PR TITLE
Update service bus SDK to T2

### PR DESCRIPTION
**Pull request by wantedfast**

### CODE REVIEW

File: Common/Utilities.cs [hunk 1]
```diff
@@ -557,2637 +559,2638 @@ public static void Print(IServiceBusNamespace serviceBusNamespace)
...
-            Utilities.Log(new StringBuilder().Append("Availability Set: ").Append(resource.Id)
-                .Append("Name: ").Append(resource.Name)
-                .Append("\n\tResource group: ").Append(resource.ResourceGroupName)
-                .Append("\n\tRegion: ").Append(resource.Region)
-                .Append("\n\tTags: ").Append(FormatDictionary(resource.Tags))
-                .Append("\n\tFault domain count: ").Append(resource.FaultDomainCount)
-                .Append("\n\tUpdate domain count: ").Append(resource.UpdateDomainCount)
-                .ToString());
-        }
+        //public static void PrintVirtualMachineCustomImage(IVirtualMachineCustomImage image)
```
Comment (wantedfast) [inline Common/Utilities.cs:562]: I commented out some useless code of this project now, and wondering if it is ok to **delete** them? 
This Utilties.cs file exist in many samples  (For example, all Service Bus sample project has this file). But like this project, not of all code needed for each project.  Some project just need two or three methods in Utilties.cs file.

File: Common/Utilities.cs [hunk 2]
```diff
@@ -3108,37 +3109,37 @@ public static string GetCertificatePath(string certificateName)
...
-                    var queueClient = new QueueClient(connectionString, queueName, ReceiveMode.PeekLock);
-                    queueClient.SendAsync(new Message(Encoding.UTF8.GetBytes(message))).Wait();
-                    queueClient.Close();
-                }
-                catch (Exception)
-                {
-                }
-            }
-        }
+        //public static void SendMessageToQueue(string connectionString, string queueName, string message)
```
Comment (wantedfast) [inline Common/Utilities.cs]: I commented out some useless code of this project now (there is an example), and wondering if it is ok to delete them?
This Utilties.cs file exist in many samples (For example, all Service Bus sample project has this file). But like this project, not of all code needed for each project. Some project just need two or three methods in Utilties.cs file.

File: Common/Utilities.cs [hunk 3]
```diff
@@ -2804,14 +2793,10 @@ public static void CreateCertificate(string domainName, string pfxName, string c
...
             {
-                //File.Copy(
-                //    Path.Combine(Utilities.ProjectPath, "Asset", "SampleTestCertificate.pfx"),
-                //    Path.Combine(Utilities.ProjectPath, "Asset", pfxName),
-                //    overwrite: true);
             }
         }
 
-        public static void UploadFileToWebApp(IPublishingProfile profile, string filePath, string fileName = null)
+        public static async Task UploadFileToWebAppAsync(IPublishingProfile profile, string filePath, string fileName = null)
```
Comment (FrankieTF) [inline Common/Utilities.cs:2799]: Heaths has reviewed this in another PR-[Azure-Samples/key-vault-dotnet-manage-key-vaults#8](https://github.com/Azure-Samples/key-vault-dotnet-manage-key-vaults/pull/8)
we decide to use await for async code, followed by .ConfigureAwait(false).

**Comment by jongio** [comment]
@jsquire - can you please review?

**Comment by jsquire** [comment]
I'm going to avoid going through a lot of the detail here, as I think this sample mirrors the feedback that I left in another.     Please see my thoughts in [this](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Messaging.ServiceBus_7.0.0/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample07_CrudOperations.md) sample.   To reflect the most important bits:

> The sample is unfocused;  it's not sure whether it wants to show off a management library or a client library.   We should be working against a tight focus and adapting the sample to that purpose.   In either case, we should not be using `Fluent` as it is no longer under active development and is only receiving critical fixes.
>
> If this sample is attempting to demonstrate "how do you create Service Bus resources using the management libraries" then we should strip out the `Azure.Messaging.ServiceBus` use and wait to update until `Azure.ResourceManager.ServiceBus` is available.
>
> If this sample is attempting to demonstrate "how do I use the administration features of `Azure.Messaging.ServiceBus` to create resources like queues and topics in my existing Service Bus namespace" then we should strip out all Fluent and other management use,  bootstrap using an ARM template to create the namespace and demonstrate the administration client similar to how this [sample](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Messaging.ServiceBus_7.0.0/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample07_CrudOperations.md) does.

@jongio: You may want to weigh in here, as I think this is salient to both the core content of the sample as well as the scheduling. Thoughts?

**Review by jsquire — COMMENTED** [review]
Each of the samples in this group are highly redundant.  I don't believe there is much value offered.  If anything, I'd recommend that we improve one with the scenarios that we want to highlight and retire the others.

We'll want to determine the focus of the sample and path forward before doing a deeper review.

**Comment by v-xuto** [comment]
@jsquire - Could you please help us close this PR? Because Wantedfast has resigned and we don't have permissions to modify this PR. We will create a new PR to replace it later.

**Comment by jsquire** [comment]
@CamSoper:  Would you be so kind as to close this out or suggest someone that I could follow-up with?

**Comment by CamSoper** [comment]
@jsquire I don't seem to have any rights to this repo.  [Looks to me](https://repos.opensource.microsoft.com/orgs/Azure-Samples/repos/service-bus-dotnet-manage-publish-subscribe-with-advanced-features) as if you need @selvasingh or @yungezz or someone on their teams.

I'm going to copy/paste this comment on the PRs you needed closed to tag them in on each.

**Comment by v-yilinhu** [comment]
PR https://github.com/Azure-Samples/service-bus-dotnet-manage-publish-subscribe-with-advanced-features/pull/3 replaces this PR https://github.com/Azure-Samples/service-bus-dotnet-manage-publish-subscribe-with-advanced-features/pull/2, so we can close this PR.

**Pull request closed without merging**